README.md

```diff
@@ -39,6 +39,24 @@ In the example below, we're showing a hypothetical project graph and what projec
 
 ## Installation
 
+```groovy
+// settings.gradle(.kts)
+pluginManagement {
+  repositories {
+    mavenCentral()
+    gradlePluginPortal()
+  }
+}
+
+plugins {
```

Comment:
Need to clarify that, while the `pluginManagement` block is in settings.gradle, this plugins directive is in the root build.gradle file.